pkg/es/reader/resolveIndexHandler.go

```diff
@@ -70,6 +75,9 @@ func ExpandAndReturnIndexNames(indexPattern string, allVirtualTableNames map[str
 		}
 
 	} else {
+		if isIndexExcluded(indexPattern) {
```

Comment:
remove this part. We want to exclude internal indexes only when we pass "*" as index value. If we send specific index name (traces/red-traces/service-dependency) we should search against the exact index names.

Reply:
This is true just for the API part that calls resolveIndexHandler.go or for the UI part [here](https://github.com/ashutoxh/siglens/blob/develop/pkg/virtualtable/virtualtable.go#L606) as well?

Reply:
Please see this https://github.com/siglens/siglens/pull/119/files#r1403638696